book/service_container.rst

```diff
@@ -221,30 +221,30 @@ straightforward. Parameters make defining services more organized and flexible:
                 http://symfony.com/schema/dic/services/services-1.0.xsd">
 
             <parameters>
-                <parameter key="my_mailer.transport">sendmail</parameter>
+                <parameter key="app.mailer.transport">sendmail</parameter>
             </parameters>
 
             <services>
-                <service id="my_mailer" class="Acme\HelloBundle\Mailer">
-                    <argument>%my_mailer.transport%</argument>
+                <service id="app.mailer" class="AppBundle\Mailer">
+                    <argument>%app.mailer.transport%</argument>
                 </service>
             </services>
         </container>
 
     .. code-block:: php
 
-        // app/config/config.php
+        // app/config/services.php
         use Symfony\Component\DependencyInjection\Definition;
 
-        $container->setParameter('my_mailer.transport', 'sendmail');
+        $container->setParameter('app.mailer.transport', 'sendmail');
 
-        $container->setDefinition('my_mailer', new Definition(
-            'Acme\HelloBundle\Mailer',
-            array('%my_mailer.transport%')
+        $container->setDefinition('app.mailer', new Definition(
+            'AppBundle\Mailer',
+            array('%app.mailer.transport%')
         ));
 
 The end result is exactly the same as before - the difference is only in
-*how* you defined the service. By surrounding the ``my_mailer.transport``
+*how* you defined the service. By surrounding the ``app.mailer.transport``
```

Comment:
`By surrounding ... in percent signs` -> `By wrapping/enclosing ... with percent signs` ?